__tests__/src/index.spec.ts

```diff
@@ -1830,10 +1888,11 @@ describe("flatpickr", () => {
       it("doesn't misfire", () => {
         let timesFired = 0;
         const fp = createInstance({
+          allowInput: true,
           onChange: () => timesFired++,
         });
         fp._input.focus();
-        document.body.focus();
+        simulate("blur", fp._input);
```

Comment:
This needed to be changed, otherwise if I make an actual change `onChange` won't be fired.